c2corg_ui/static/js/documentediting.js

```diff
@@ -80,9 +81,9 @@ app.DocumentEditingController = function($scope, $element, $attrs,
 
   /**
    * @type {app.Authentication}
-   * @private
+   * @export
```

Comment:
You could use `@public`. The underscore is only required for private attributes/methods.